test/functional/wallet_listtransactions.py

```diff
@@ -210,5 +212,57 @@ def get_unconfirmed_utxo_entry(node, txid_to_match):
         assert_equal(self.nodes[0].gettransaction(txid_3b)["bip125-replaceable"], "no")
         assert_equal(self.nodes[0].gettransaction(txid_4)["bip125-replaceable"], "unknown")
 
+    def create_and_send_transaction(self, utxo, address, amt, feeRate):
+        psbtx = self.nodes[0].walletcreatefundedpsbt([{"txid": utxo['txid'], "vout": utxo['vout']}],
+                                                      {address: amt},
+                                                      0,
+                                                      {"replaceable":True, "feeRate":feeRate})['psbt']
```

Comment:
It might be clearer to use named arguments here, e.g.:
```python
+        psbtx = self.nodes[0].walletcreatefundedpsbt(
+            inputs=[{"txid": utxo["txid"], "vout": utxo["vout"]}],
+            outputs={address: amount},
+            locktime=0,
+            options={"replaceable": True, "fee_rate": fee_rate},
+        )["psbt"]
```